### PR TITLE
Add a note about quarkus.ssl.native in the mp.jwt.verify.publickey.location description

### LINKED ISSUE #3502: Specify in the JWT guide that using HTTPS mp.jwt.verify.publickey.location requires quarkus.ssl.native in the native mode
**Description**
The JWT documentation is missing this information
**Implementation ideas**
Update the docs as suggested in the issue title

### PR DESCRIPTION
**Pull request by sberyozkin**
Fixes #3502

### CODE REVIEW

File: docs/src/main/asciidoc/jwt-guide.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ secured access to the JAX-RS endpoints.
 |quarkus.smallrye-jwt.realm-name|Quarkus-JWT|Name to use for security realm.
 |quarkus.smallrye-jwt.auth-mechanism|MP-JWT|Name to use for authentication mechanism
 |mp.jwt.verify.publickey|none|The `mp.jwt.verify.publickey` config property allows the Public Key text itself to be supplied as a string.  The Public Key will be parsed from the supplied string in the order defined in section <<Supported Public Key Formats>>.
-|mp.jwt.verify.publickey.location|none|Config property allows for an external or internal location of Public Key to be specified.  The value may be a relative path or a URL.
+|mp.jwt.verify.publickey.location|none|Config property allows for an external or internal location of Public Key to be specified.  The value may be a relative path or a URL. If the value points to an HTTPS based JWK set then for it to work in the native mode a `quarkus.ssl.native` property must also be set to `true`, see link:native-and-ssl-guide.html[Using SSL With Native Executables].
```
Comment (gsmet) [inline docs/src/main/asciidoc/jwt-guide.adoc]:
```suggestion
|mp.jwt.verify.publickey.location|none|Config property allows for an external or internal location of Public Key to be specified.  The value may be a relative path or a URL. If the value points to an HTTPS based JWK set then, for it to work in native mode, the `quarkus.ssl.native` property must also be set to `true`, see link:native-and-ssl-guide.html[Using SSL With Native Executables] for more details.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
I suggested a few improvements, otherwise it looks good, thanks.

**Comment by sberyozkin** [comment]
@gsmet sure, yes it has to be precise in the docs, just merged your suggestion directly from this page, thanks

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Squashed and merged, thanks.

**Pull request merged**